Source/OxyPlot.Wpf/XpsExporter.cs

```diff
@@ -9,7 +9,7 @@
 
 namespace OxyPlot.Wpf
 {
-#if !NET40
```

Comment:
To review - I'm not sure replacing `#if !NET40` by `#if !NETFRAMEWORK` is correct

Reply:
Am I wrong to think we can just remove this check altogether? (not checked, in the middle of other things)

Reply:
fair point, checking that

Reply:
I think it's better to leave it like that, `System.Windows.Xps` is not available in `net452` but is in `netcoreapp3.1`. So it seems checking for `#if !NETFRAMEWORK` is correct